src/data.js

```diff
@@ -241,6 +241,17 @@ module.exports = [
       'Teacher',
     ],
   },
+  {
+    name: 'Jay Tyrrell',
+    description: 'Full Stack Developer',
+    url: 'https://jaytyrrell.co/uses/',
```

Comment:
Is it my internet or is your /uses page empty?

Reply:
That's strange. My /uses page is filled out. Could you try again? Are you trying with https?

Reply:
I've checked it, it's filled out